pkg/kubelet/OWNERS

```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - rphillips
+  - sjenning
+  - mrunalp
+
+# sub-package approvers for "UPSTREAM: <PR>: ..." changes that merged upstream
```

Comment:
Add an indication that these approvers are also present in the upstream owners file please.